components/story-inquiry-page/StoryInquiryPage.js

```diff
@@ -0,0 +1,190 @@
+    story_concept_popup_title: popupTitle,
+    story_concept_popup_description: popupDescription,
+    story_concept_popup_image: popupImage,
+    resource_links_subtitle: resourceLinksSubtitle,
+    resource_links_description: resourceLinksDescription,
+    faq_title: faqTitle,
+    author_testimonies_title: title,
+    author_testimonies_description: description,
+    author_testimonies_image: quoteImage,
+    story_submission_confirmation_title_regular: confirmationTitleRegular,
+    story_submission_confirmation_title_italic: confirmationTitleItalic,
+    story_submission_confirmation_description: confirmationDescription,
+    story_submission_confirmation_image: confirmationImage,
+    spark_arrow_description_text: sparkArrowDescriptionText,
+    spark_arrow_faq_text: sparkArrowFaqText,
+    spark_arrow_author_testimonies_text: sparkArrowAuthorTestimoniesText,
+    footer_image: footerImage,
+    story_inquiry_top_image: storyInquiryTopImage,
+    story_inquiry_side_image_1: storyInquirySideImage1,
+    story_inquiry_side_image_2: storyInquirySideImage2,
+    story_inquiry_side_image_3: storyInquirySideImage3,
+  } = storyInquiryPageData;
+
+  const storyConceptFaqData = {
+    faqTitle,
+    questionsAndAnswers,
+  };
+
+  const storySubmissionConfirmationData = {
+    confirmationTitleRegular,
+    confirmationTitleItalic,
+    confirmationDescription,
+    confirmationImage,
+  };
+
+  const quotes = authorTestimonies.map((q) =>
+    getString(q.author_testimony).length > 154 ?
+      getString(q.author_testimony).slice(0, 154) + "..." :
+      getString(q.author_testimony));
+
+  const authors = authorTestimonies.map((m) => getString(m.author_name));
+
+  const authorTestimoniesData = {
+    title,
+    description,
+    quotes,
+    quoteImage,
+  };
+
+  const boxLinkFooter = { img: footerImage, data: boxLinkDataFooter };
+
+  const storyInquiryFormData = {
+    authorInformationSubtitle,
+    authorSignatureSubtitle,
+    authorSignatureDescription,
+    socialInformationSubtitle,
+    venmoMoreInfoSubtitle,
+    venmoMoreInfoDescription,
+    storyConceptSubtitle,
+    storyConceptDescription,
+    storyConceptBoxSubtitle,
+    resourceLinksSubtitle,
+    resourceLinksDescription,
+  };
+
+  const storyConceptPopupData = {
+    popupTitle,
+    popupDescription,
+    popupImage,
+  };
+
+  return (
+    <>
+      <Navigation navigationData={navigationData} />
+      <H1StyledContainer>
+        <H1Styled
+          regular={storySubmissionTitleRegular}
+          italicized={storySubmissionTitleItalic}
+          storyInquiry={true}
+        />
+        <SquiggleWavy src={icons.SQUIGGLE_WAVY} alt="wavy squiggle underline" />
+      </H1StyledContainer>
+
+      <PageDescription
+        description={storySubmissionDescription}
+        arrowText={sparkArrowDescriptionText}
+        img={storyInquiryTopImage}
+      />
+      <TopSparkArrowContainer>
+        <LongSparkArrow arrowText={sparkArrowFaqText} />
```

Comment:
please add functionality to all long spark arrows, example can be seen [here](https://github.com/Scout-NU/thealleyway-site/pull/123) lmk if you need help